### ISSUE TITLE
Download option not working after deployment

### ISSUE THREAD

**Issue by hrishibawane** [hrishibawane/ipfs-pastebin#1]
Download doesn't work after deployment, but works locally

**Comment by Zechro** [comment]
I am very new to Git and dont understand exactly how to use it. I solved your issue but im having trouble figuring out how to commit/merge the code. Do you have to assign me?

**Comment by hrishibawane** [comment]
Hi @Zechro. Well you must create a new branch and push your changes on that branch. Then create a Pull Request. I will then merge your code into the master branch.
I strongly suggest you go through the following link to understand exactly **how to do these steps**:
[https://www.dataschool.io/how-to-contribute-on-github/](https://www.dataschool.io/how-to-contribute-on-github/)

**Comment by Rogue-Halo** [comment]
Out of interest. what was the fix?

**Comment by Zechro** [comment]
In the code, you were creating a txt file named pastebin, but you were
creating it directly to the local downloads folder. The server has no way
to access the downloads folder so you must send a file through node with
specific endpoint. Thats why it worked locally, you had access to the
downloads folder since it was running locally too, and didnt work when
deployed.

To fix the problem I just  A. created a new downloads endpoint which
creates the pastebin file, then sends it to the user, and B. removed the
button from the form so it actually no longer has functionality and instead
gave it an onclick attribute which opens a new window to the downloads link.



OH I just realized I made a mistake

The redirct link in the EJS file will send the user to localhost. This wont
work in deployment. All I need to know is the url it will be hosted on to
fix it. Is that still gonna be heroku?
